content/PGP/Card_edit.adoc

```diff
@@ -12,11 +12,18 @@ To configure the OpenPGP application on a YubiKey you can use GnuPG.
 
 == Prerequisites
 
-Make sure you have a recent YubiKey with firmware version 3.1.8 or
-later; use "lsusb -v" to find out.
+Installed the link:https://developers.yubico.com/yubikey-personalization/Manuals/ykpersonalize.1.html[Yubikey personalization tool].
 
-Make sure the device is in OTP/CCID or CCID mode, use the YubiKey
-personalization tool "ykpersonalize -m82" to switch.
+Verify that your YubiKey has a firmware version 3.1.8 or later.
+
+  $ lsusb -v
+  ...
+
+Make sure that your device is in OTP/CCID or CCID mode, you can use ykpersonalize to switch
+to OTP/U2F/CCID:
+
+  $ ykpersonalize -m86
```

Comment:
The 8 is actually redundant for all modes except 1 (CCID only) and early NEO devices in mode 2 (hence the 82 value). Setting mode 86 should be equivalent to setting mode 6, always.